### PR TITLE
OCP4: Add control response for AC-12(1)

### PR DESCRIPTION
**Pull request by Vincent056**
Added control response for AC-12(1), section b is blocked by an RFE: https://issues.redhat.com/browse/RFE-2645

### CODE REVIEW

File: controls/nist_ocp4.yml [hunk 1]
```diff
@@ -1463,16 +1463,20 @@ controls:
...
 - id: AC-12(1)
-  status: pending
+  status: partial
   notes: |-
-    Section a: A control response is planned. Progress can be tracked via:
+    OpenShift provides a logout capability for user-initiated communications sessions whenever
+    authentication is used to gain access to the system when the default kubeadmin account is disabled.
+    
+    The OpenShift does not have the logout capability when cert is being used with the default kubeadmin account,
+    therefore we need to check if kubeadmin account is disabled.
```
Comment (jhrozek) [inline controls/nist_ocp4.yml:1472]: What do you think about using the text about certificate logins and revocation we have here: https://github.com/JAORMX/content/blob/be396cf36803f9890d6947d9688a74b3a9f535d7/applications/openshift/authentication/idp_is_configured/rule.yml#L42 ?

File: controls/nist_ocp4.yml [hunk 2]
```diff
@@ -1463,16 +1463,20 @@ controls:
   - high
   - moderate
 - id: AC-12(1)
-  status: pending
+  status: partial
```
Comment (jhrozek) [inline controls/nist_ocp4.yml:1466]: This is marked as partial becuase there's an RFE right?

**Review by jhrozek — COMMENTED** [review]
Looks good! One nit and one question inline

File: controls/nist_ocp4.yml (hunk 1 above)
Comment (Vincent056) [reply controls/nist_ocp4.yml:1472]: Thanks for the suggestion, I will update the content there

File: controls/nist_ocp4.yml (hunk 2 above)
Comment (Vincent056) [reply controls/nist_ocp4.yml:1466]: Yes, that's right

**Review by jhrozek — APPROVED** [review]
/lgtm

**Pull request merged**